src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -66,12 +65,12 @@ public ObjectionService(ObjectionRepository objectionRepository,
     }
 
     @Override
-    public String createObjection(String requestId, String companyNumber, String ericUserId, String ericUserDetails) throws Exception{
+    public String createObjection(String requestId, String companyNumber, String ericUserId, String ericUserDetails) {
```

Comment:
Does this not complain that it no longer matches the interface? As the interface createObjection throws Exception
Though if it doesn't then I agree we should remove, but i'd say remove from interface too?

Reply:
The IDE highlights it as well in the interface as being too generic, i must have missed that one as well.